src/components/EnemyCollision.js

```diff
@@ -0,0 +1,40 @@
+import AFRAME from 'aframe';
+
+AFRAME.registerComponent('enemy-collision', {
+  play: function () {
+    let hit = false
+    // check for collision
+    this.el.addEventListener('collide', function (e) {
+        setTimeout(function() {
+            //if we wanted just enemies to be hit we could make that a component and then only hit enemies
+            //make sure this is the first collision with something besides the player character
+            if (e.detail.body.el != document.querySelector('#playerChar') && e.detail.body.el!= null && hit == false) {
```

Comment:
we should just probably add an id to the player and if collider in the callback is on the player just ignore it

Reply:
I'm a little confused on what you mean by this. Right now, only the projectiles actually use the collision component (structuring it this way makes it easier because we don't have to account for normal collisions such as the player touching the floor) and the check in line 11 is based on the player character's id. Do you mean you want me to do that check before the setTimeout in the code (just inside the event listener)?